docs/netserver/config/data.md

```diff
@@ -1,65 +1,118 @@
 
 | Name | Description |
 |---|---|
 | Mode | The mode of the session: Thread, Context, HttpContext, Process.<br>Maps to a class name that provides session storage. |
-| ReauthenticateOnDeserialization | A full re-authentication is carried out each time the session is de-serialized (e.g. from the session server). |
+| ReauthenticateOnDeserialization | A full re-authentication is carried out each time the session is de-serialized (such as from the session server). |
+
+Each session option determines where in memory the authenticated session will be available from.
+
+| Mode | Availability of authenticated SoSession |
+|---|---|
+| Process | to all synchronization contexts and threads |
+| Context | only to routines passed to the same synchronization context |
+| Thread  | only on that thread - if another non-authenticated thread tried to use the SuperOffice API to get data, NetServer would throw an exception. |
+
+* **Thread:** Every session requires the [suspend][1] and continue methods to be called for each query task. Here, each thread executed in NetServer will have sessions that are stored in the implication of different threads and have different session values stored in them. If you want to suspend a session, you must call the suspend method which returns a string with the session values. Should you wish to continue that session, you must call the continue method and pass the string with session values in it as a parameter. The session is stored in a thread-static manner.  Each executing thread has a separate instance of the session. If the session is accessed on a different thread, it will contain a different value. For further reference, see `System.ThreadStaticAttribute`.
+
+* **Context:** Your session requires only authentication and a closure. Here, once you authenticate a session, it will be for the lifetime of the session. It is not necessary to call continue and suspend methods. The session values for this configuration will be stored in a context-static manner. For further reference, see `System.ContextStaticAttribute`.
+
+* **Process:** Session is stored in a process-static manner. This is the normal option for Windows applications.
+
+* **HttpContext:** Recommended in ASP.NET applications. Added to prevent ASP.NET thread switches from changing identities in the middle of an ASP.net request.
 
-When in Mode **Thread**, every session requires the [suspend][1] and continue methods to be called for each query task. Here, each thread executed in NetServer will have sessions that are stored in the implication of different threads and have different session values stored in them. If you want to suspend a session, you must call the suspend method which returns a string with the session values. Should you wish to continue that session, you must call the continue method and pass the string with session values in it as a parameter.
+* **PartnerHttpContext:** Helper class for `SuperOffice.DevNet.Online.Login`.
```

Comment:
Remove PartnerHttpContext ... It's not build-in and not an option out of the box...